src/components/chats/ChatMessage.tsx

```diff
@@ -0,0 +1,89 @@
+import styled, { css } from "styled-components"
+import { Text } from "components/text/Text"
+import { useTheme } from "styles/theme/useTheme"
+
+export type ChatMessageProps = {
+  message: string
+  dateTime: string
+  icon?: React.ReactNode
+}
+
+export const ChatMessage: React.FC<ChatMessageProps> = ({
+  message,
+  dateTime,
+  icon
+}) => {
+
+  const theme = useTheme()
+
+  return (
+    <Container>
+      <MessageContainer
+        css={`
+          flex-direction: ${icon ? "row-reverse" : "row"};
+        `}
+      >
+        <MessageText
+          css={`
+           margin-right: ${icon ? "0px" : "10px"};
```

Comment:
You don't have to change these margin css rules. However, since this component is nested in a flex container, you can just add a css `gap` rule to the parent and not have to determine whether or not you need to add a right margin here and for the IconContainer.

The size of the gap/margin is 14px / 1rem. You can grab the `SM` property from the predefined SPACINGS object, which should have the correct measurement.

Reply:
Will do that